### ISSUE TITLE
Volume very quiet for me

### ISSUE THREAD

**Issue by 2405anthony** [MGraefe/RP-Soundboard#99]
Hello. When I play a Sound on the Soundboard for I cant hear it load enough, but other hear it loud. When I adjust it to the volume that I can hear it good enough, its earpain for the other. Any suggestion?

**Comment by MGraefe** [comment]
This came up a lot, although I am not sure how to solve this. Most likely it has something to do with Teamspeaks automatic gain control. All I can say is that I add the same volume on both the microphone channel (which goes to your friends) as well as the speaker channel (which goes to your....well...ears).

Probably one could build an option to separate the volume sliders into microphone and speakers. I sadly don't have any time for this at the moment but I would gladly accept any pull request. The only requirement I have that this doesn't complicate the volume sliders for new or existing users that don't need this feature.

**Comment by DooubleTap** [comment]
Yeah, thatès the ONLY issue with this soundboard, i enjoy spending time with friends on some fivem rp and to listen to music, but for me it soulds like a 1/10 when for them its 10/10. This would be an amazing feature to have (separate volume slider for what we hear.
